docs/source/whats-new.rst

```diff
@@ -26,15 +26,16 @@ New Features
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
-- Renamed `DataTree.lineage` to `DataTree.parents` to match `pathlib` vocabulary
-  (:issue:`283`, :pull:`286`)
 - Change default write mode of :py:meth:`DataTree.to_zarr` to ``'w-'`` to match ``xarray``
   default and prevent accidental directory overwrites. (:issue:`274`, :pull:`275`)
   By `Sam Levang <https://github.com/slevang>`_.
 
 Deprecations
 ~~~~~~~~~~~~
 
+- Renamed `DataTree.lineage` to `DataTree.parents` to match `pathlib` vocabulary
+  (:issue:`283`, :pull:`286`). `lineage` is now deprecated and use of `parents` is encouraged.
+
```

Comment:
You are welcome to credit yourself for the contribution here if you want to 🙂 See other whats-new entries for how to format it to link to your github profile.